### ISSUE TITLE
Compound Motor Action Potential Abnormality

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5935]
**Preferred term label:** Compound Motor Action Potential Abnormality

**Synonyms** CMAP Abnormality

**Definition (free text, please give PubMed ID)** Abnormally high or low compound action potential reading (-Pk Amps), PMID:26744834

**Parent term (use hpo.jax.org/app)** Abnormal motor nerve conduction velocity HP:0040131

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
